README.md

```diff
@@ -3,7 +3,7 @@
 
 _Cloud Container Attack Tool (CCAT) is a tool for testing security of container environments._
 
-<img src="https://raw.githubusercontent.com/RhinoSecurityLabs/ccat/master/docs/images/ccat_main_menu.png" width="1000"/>
+<img src="https://raw.githubusercontent.com/RhinoSecurityLabs/ccat/feature/gcp/docs/images/ccat_new_main_menu.png" width="1000"/>
```

Comment:
You can use relative URLs here so you don't have to read a picture from a different branch

Should just be able to do `src="docs/images/ccat_new_main_menu.png"`

Reply:
That is a good way to do that but docker hub's readme can not read from this path b/c of that reason I am using githubusercontent path.